suites/squid/common/regression/ecpool-replica1-deploy-and-configure.yaml

```diff
@@ -0,0 +1,132 @@
+# Deployment for Ceph cluster across 5 nodes & 1 client
+# conf: conf/squid/common/5-node-1client-ecpool-replica1.yaml
+tests:
+  - test:
+      name: setup install pre-requisites
+      desc: Setup phase to deploy the required pre-requisites for running the tests.
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+      name: cluster deployment
+      desc: Execute the cluster deployment workflow.
+      module: test_cephadm.py
+      polarion-id:
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              base_cmd_args:
+                verbose: true
+              args:
+                registry-url: registry.redhat.io
+                mon-ip: node1
+                orphan-initial-daemons: true
+
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+
+          - config:
+              command: apply
+              service: mgr
+              args:
+                placement:
+                  label: mgr
+
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  label: mon
+
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+
+          - config:
+              command: shell
+              args:          # arguments to ceph orch
+                - ceph
+                - fs
+                - volume
+                - create
+                - cephfs
+
+          - config:
+              command: apply
+              service: rgw
+              pos_args:
+                - rgw.1
+              args:
+                placement:
+                  label: rgw
+
+          - config:
+              command: apply
+              service: mds
+              base_cmd_args:          # arguments to ceph orch
+                verbose: true
+              pos_args:
+                - cephfs              # name of the filesystem
+              args:
+                placement:
+                  nodes:
+                    - node2
+                    - node3
+                  limit: 2            # no of daemons
+                  sep: " "            # separator to be used for placements
+      destroy-cluster: false
+      abort-on-fail: true
+
+  - test:
+      name: Configure client admin
+      desc: Configures client admin node on cluster
+      module: test_client.py
+      polarion-id:
+      config:
+        command: add
+        id: client.1                      # client Id (<type>.<Id>)
+        node: node5                       # client node
```

Comment:
Client needs to be configured on node6, not on backup node (node5)
Please refer: `conf/squid/rados/4-node-ec-cluster-1-client.yaml`